### ISSUE TITLE
Crash when switching chats

### ISSUE THREAD

**Issue by no-boot-device** [CreeperHost/CreeperHostGui#94]
```
Time: 8/2/19 4:33 AM
Description: Updating screen events

java.lang.NullPointerException: Updating screen events
	at net.creeperhost.minetogether.gui.chat.GuiMTChat.formatLine(GuiMTChat.java:533)
	at net.creeperhost.minetogether.gui.chat.ingame.GuiNewChatOurs.rebuildChat(GuiNewChatOurs.java:246)
	at net.creeperhost.minetogether.gui.chat.ingame.GuiChatOurs.func_146284_a(GuiChatOurs.java:311)
	at net.minecraft.client.gui.GuiScreen.func_73864_a(GuiScreen.java:443)
	at net.minecraft.client.gui.GuiChat.func_73864_a(SourceFile:128)
	at net.creeperhost.minetogether.gui.chat.ingame.GuiChatOurs.func_73864_a(GuiChatOurs.java:154)
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:533)
	at net.minecraft.client.gui.GuiChat.func_146274_d(SourceFile:100)
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:501)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1759)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1098)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

Full log: https://files.smooch.computer/x/crash-2019-08-02_04.33.55-client.txt

**Comment by no-boot-device** [comment]
I can no longer reproduce this as I've been "permanently banned (no reason)". Hm. Anyway, if anyone else wants to try and reproduce this and narrow it down, they can go ahead.

**Comment by CH-Paul-T** [comment]
Previous comment removed as it provided no additional information for the bug report.

**Comment by gigabit101** [comment]
This should be resolved as of 2.2.0-RC6
